src/Orc.Controls/Controls/StepBar/Converters/CommandCanExecuteToCursorConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Orc.Controls
+{
+    using System;
+    using Catel;
+    using Catel.MVVM.Converters;
+
+    public class CommandCanExecuteToCursorConverter : ValueConverterBase
+    {
+        public CommandCanExecuteToCursorConverter()
+        {
+        }
+
+        protected override object Convert(object value, Type targetType, object parameter)
+        {
+            if (value is Catel.MVVM.TaskCommand<IStepBarItem> command)
```

Comment:
I am not sure how this will work combined with the canexecute state of the command. Maybe we could "abuse" a (transparent) button which we will hide with converter to self.IsEnabled.

The button will take care of the event subscriptions of the command and re-evaluate automatically if required.

Reply:
Yes, this is a great idea, I've pushed some changes now :)